packages/api/lambdas/publish-granules.js

```diff
@@ -14,8 +14,31 @@ const handler = async (event) => {
 
   const promisedPublishEvents = event.Records.map(
     (record) => {
-      const granule = attr.unwrap(record.dynamodb.NewImage);
-      return publishSnsMessage(topicArn, granule);
+      const message = {};
+      const newImage = attr.unwrap(record.dynamodb.NewImage);
+      const oldImage = attr.unwrap(record.dynamodb.OldImage);
+      switch (record.eventName) {
+      case 'INSERT': {
```

Comment:
You don't need the squigly-brackets in these case statements: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch